android/fastlane/README.md

```diff
@@ -0,0 +1,29 @@
+fastlane documentation
+================
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
+
+```
+xcode-select --install
+```
+
+Install _fastlane_ using
+```
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew cask install fastlane`
+
+# Available Actions
+## Android
+### android alpha
+```
+fastlane android alpha
+```
+Deploy a new alpha version to the Google Play Store
+
+----
+
+This README.md is auto-generated and will be re-generated every time [fastlane](https://fastlane.tools) is run.
```

Comment:
Since it is auto-generated every time fastlane is run,  should this be part of git ?